frontend/src/environments/environment.development.ts

```diff
@@ -1,8 +1,7 @@
 export const environment = {
     BACKEND_API_URL: 'http://localhost:8000/v1',
     DEFAULT_PAGE_SIZE: 10,
-    AIOD_KEYCLOAK_URL: 'http://ec2-3-79-133-150.eu-central-1.compute.amazonaws.com/aiod-auth',
-    // AIOD_KEYCLOAK_URL: 'https://aiod-dev.i3a.es/aiod-aunpmth',
+    AIOD_KEYCLOAK_URL: 'https://aiod-dev.i3a.es/aiod-auth',
```

Comment:
Please revert these changes for now. A separate PR is on its way for the overall FE deployment